### PR TITLE
Unify both cypress and baobab into main

### PR DESCRIPTION
**Pull request by jay-kim-kor**

### CODE REVIEW

**Comment by nick-bisonai** [comment]
When deploying new image, I used to update 2 parts
- `.global.image.tag` in `values.yaml`
- `.appVersion` in `Chart.yaml`

But now only `values.yaml` has separated version for baobab and cypress. Wouldn't I have to update `Chart.yaml` file after this update?

**Comment by jay-kim-kor** [comment]
> When deploying new image, I used to update 2 parts
> 
> * `.global.image.tag` in `values.yaml`
> * `.appVersion` in `Chart.yaml`
> 
> But now only `values.yaml` has separated version for baobab and cypress. Wouldn't I have to update `Chart.yaml` file after this update?

I recommend to update appVersion in Chart.yaml when update image tag in `values.cypress.yaml`.

**Comment by nick-bisonai** [comment]
> > When deploying new image, I used to update 2 parts
> > 
> > * `.global.image.tag` in `values.yaml`
> > * `.appVersion` in `Chart.yaml`
> > 
> > But now only `values.yaml` has separated version for baobab and cypress. Wouldn't I have to update `Chart.yaml` file after this update?
> 
> I recommend to update appVersion in Chart.yaml when update image tag in `values.cypress.yaml`.

should I set Chart.yaml up to newer version between baobab and cypress?

**Review by nick-bisonai — APPROVED** [review]
lgtm!

File: .husky/post-commit
Comment (jay-kim-kor) [inline .husky/post-commit:1]: Resolve #286

**Pull request merged**